doc/guides/CustomStartupGuide.md

```diff
@@ -546,6 +550,9 @@ Follow these instructions to update devices.
 
 1. Get the profile token by executing the `GetProfiles` command:
 
+
+   >NOTE: Make sure to replace `Camera001` in all the commands below, with the deviceName returned in the "Verify device(s) have been successfully added to core-metadata" step above.  
```

Comment:
This is repetitive of a note above, I think this is a better place for it.

Reply:
I have kept this note and removed the one from the comment above